ffi/pic.lua

```diff
@@ -221,6 +221,37 @@ function Pic.openGIFDocument(filename)
     return GifDocument:new{giffile = giffile}
 end
 
+function Pic.openGIFDocumentFromData(data, size)
+    -- Create GIF from data pointer (from https://github.com/luapower/giflib)
```

Comment:
Oh, I recently came across LuaPower. It looks pretty neat. They also have a harfbuzz FFI binding which I hope I recently linked to somewhere relevant.

Other stuff of similar potential interest includes http://torch.ch/ and https://github.com/Planimeter/lgf